modules/spring-config/src/main/java/org/mule/config/spring/MuleConfigurationConfigurator.java

```diff
@@ -84,6 +89,22 @@ public Object getObject() throws Exception
         }
     }
 
+    private void determineDefaultProcessingStrategy(DefaultMuleConfiguration defaultConfig)
+    {
+        if (config.getDefaultProcessingStrategy() != null)
+        {
+            defaultConfig.setDefaultProcessingStrategy(config.getDefaultProcessingStrategy());
+        }
+        else
+        {
+            String processingStrategyFromSystemProperty = System.getProperty(MuleProperties.MULE_DEFAULT_PROCESSING_STRATEGY);
```

Comment:
Look at responseTimeout default.  That can be configured via system property and spring.

MuleProperties.SYSTEM_PROPERTY_PREFIX + "timeout.synchronous"